File: backend/user-auth-service/src/main/java/com/tripify/user_auth_service/dto/response/UserResponse.java
package com.tripify.user_auth_service.dto.response;

public record UserResponse(
        String id,
        String name,
        String surname,
        String email,
        String profilePictureUrl,
        String phone,
        String address,
        String companyName,
        String vatNumber,
        String pec,
        String role
) {
}

File: frontend/app/src/main/java/com/tripify/tripify_android/data/UserResponse.kt
package com.tripify.tripify_android.data

data class UserResponse(
    val id: String? = null,
    val name: String?,
    val surname: String?,
    val email: String? = null,
    val profilePictureUrl: String?,
    val phone: String?,
    val address: String?,
    val companyName: String?,
    val vatNumber: String?,
    val pec: String?
)